recipes-devtools/castxml/castxml_git.bb

```diff
@@ -0,0 +1,29 @@
+SUMMARY = "C-family abstract syntax tree XML output tool."
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=3b83ef96387f14655fc854ddc3c6bd57"
+
+SRC_URI = "git://github.com/CastXML/CastXML"
+
+# 0.3.6 is the release for LLVM/Clang v11.0.0
+SRCREV = "902ac163f0291fcfc459f58691481e88c9f91dea"
+PV = "0.3.6"
```

Comment:
move this to recipe name ( call recipe as castxml_0.3.6.bb ) and you don't need it here.